pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -253,6 +253,7 @@ class Config11 {
                         'special.system'
                     ],
                     weekly : [
+                        'dev.jck',
```

Comment:
Confirming that we want this enabled on the CE spec as opposed to the OE/CRIU spec?

Reply:
Added dev.jck in default and all overridden settings in jdk11u_pipeline_config.groovy as discussed.